### PR TITLE
Displays links to active matches on separate lines

### PR DESCRIPTION
**Pull request by kwiliarty**

### CODE REVIEW

**Comment by kwiliarty** [comment]
This turned out to be very minor, indeed, but it's still an improvement, I believe.

File: app/views/qualified_opportunities/index.haml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     %p
       #{@client.full_name} is already active in at least one match:
       - @client.active_matches.each do |match|
-        %br
+        %p
```
Comment (eanders) [inline app/views/qualified_opportunities/index.haml:10]: This is fine, but I actually think the issue is just that the `=` on the line below the `%br` is indented.  Can you test and see if just outdenting that to the same depth as the %br fixes it? (Adding p tags seems extreme and may bring unwanted padding along for the ride.)

Comment (kwiliarty) [reply app/views/qualified_opportunities/index.haml:10]: Outdenting doesn't help. If I remove the p-tag in line 7 then it does. Everything then gets centered, though.

Comment (eanders) [reply app/views/qualified_opportunities/index.haml:10]: Thanks for testing.  p tag it is!

**Pull request merged**